assets/script.js

```diff
@@ -0,0 +1,203 @@
+// Declaring global variables
+var today = dayjs();
+var full_name = "";
+var index;
+var tarotDesc = [];
+var tarotName = [];
+var historySign;
+var historyName;
+var tarotID;
+
+today = today.format('MM/DD/YYYY');
```

Comment:
Try not to change around the data types of your variables.  You changed today from a dayjs object to a string.

In this instances, it's better to have 2 separate variables.  Alternately, you could just try to chain this variable when it's declared as follows

```
var today = dayjs().format('MM/DD/YYYY');
```
```